impeller/tools/impeller.gni

```diff
@@ -25,13 +25,6 @@ declare_args() {
   impeller_enable_vulkan = (is_linux || is_win || is_android ||
                             enable_unittests) && target_os != "fuchsia"
 
-  # Whether playgrounds should run with Vulkan.
-  #
-  # impeller_enable_vulkan may be true in build environments that run tests but
-  # do not have a Vulkan ICD present.
-  impeller_enable_vulkan_playgrounds =
-      (is_linux || is_win || is_android) && target_os != "fuchsia"
```

Comment:
I think you might have to do `impeller_enable_vulkan_playgrounds = impeller_enable_vulkan` for now.  You might be turning the playgrounds on without compiling in vulkan?

Reply:
I removed the ifdefs for impeller_enable_vulkan_playgrounds too. So IMPELLER_ENABLE_VULKAN is the only flag gating Vulkan playgrounds.